src/Resources/skeleton/registration/RegistrationController.tpl.php

```diff
@@ -104,11 +113,26 @@ public function register(Request $request, UserPasswordEncoderInterface $passwor
 <?php } ?>
     public function verifyUserEmail(Request $request): Response
     {
+<?php if ($verify_email_with_auth): ?>
         $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
+<?php else: ?>
+        $id = $request->get('id');
+
+        if (null === $id) {
+            return $this->redirectToRoute('app_register');
+        }
+
+        $repository = $this->manager->getRepository(<?= $user_class_name ?>::class);
```

Comment:
I can't remember: is there a reason we don't autowire the repository directly instead of the EM?

Reply:
done